### PR TITLE
libpod removal from main (phase 2)

### PR DESCRIPTION
**Pull request by baude**
this is phase 2 for the removal of libpod from main.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/logs.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"time"
 
 	"github.com/containers/libpod/cmd/podman/cliconfig"
-	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/libpod/logs"
```
Comment (TomSweeneyRedHat) [inline cmd/podman/logs.go]: +1

**Comment by TomSweeneyRedHat** [comment]
Do you have a gomod/vendor issue?

```
[21:27:22] END - [+0048s] total duration since START
cd ${GOSRC} && ./hack/tree_status.sh |& ${TIMESTAMP}
[21:27:22] START - All [+xxxx] lines that follow are relative to right now.
[+0000s] tree is dirty, please sync the vendor.conf and commit all changes.
[+0000s] 
```

**Comment by TomSweeneyRedHat** [comment]
Changes in general LGTM if you can clean up the vendor/lint fun.

**Comment by jwhonce** [comment]
Assuming vendoring is fixed.

/lgtm
/hold

**Comment by baude** [comment]
@vrothberg what do i need to do here?

**Comment by mheon** [comment]
Why is container.go growing log functions? Those either belong in
container_api.go or libpod/log.go. Ideally the latter, there are some
runtime functions mixed in there.

**Comment by baude** [comment]
> Why is container.go growing log functions? Those either belong in container_api.go or libpod/log.go. Ideally the latter, there are some runtime functions mixed in there.
> 

@mheon Im happy to make that change.  thanks

**Comment by vrothberg** [comment]
> @vrothberg what do i need to do here?

```
cd ${GOSRC} && ./hack/tree_status.sh |& ${TIMESTAMP}
[21:27:22] START - All [+xxxx] lines that follow are relative to right now.
[+0000s] tree is dirty, please sync the vendor.conf and commit all changes.
[+0000s] 
[+0000s]  M vendor/modules.txt
```

`./hack/tree_status.sh` is just a `git status`. The important part is `M vendor/modules.txt` indicating that this file is modified by `make vendor` but not committed.

You can fix it with `make vendor`, `git add ./vendor/modules.txt` and then commit/push.

**Comment by vrothberg** [comment]
LGTM

**Pull request merged**